content/api-references/funding/ACH.md

```diff
@@ -145,4 +145,100 @@ _`account_id` or `relationship_id` invalid_
 
 {{</hint>}}
 
+---
+ 
+## **Plaid Integration for Bank Transfers**
+
+We have integrated with Plaid to allow you to seamlessly link your Plaid account to Alpaca. The integration will allow your end-users to verify their account instantly through Plaid’s trusted front-end module. 
+
+Leveraging this allows you to generate Plaid Processor Tokens on behalf of your end-users, which allows Alpaca to immediately retrieve a user's bank details in order to deposit or withdraw funds on the Alpaca platform. 
+
+You can utilize your Plaid account and activate the Alpaca integration within the Plaid dashboard.
+
+The integration requires [Plaid API Keys](https://plaid.com/docs/auth/partnerships/alpaca/) 
+
+### Obtaining a Plaid Processor Token
+A Plaid processor token is used to enable Plaid integrations with partners. After a customer connects their bank using Plaid Link, a processor token can be generated at any time. Please refer to the [Plaid Processor Token](https://plaid.com/docs/auth/partnerships/alpaca/) using Alpaca page for creating a token and additional details.
+
+Exchange token
+
+```bash
+curl -X POST https://sandbox.plaid.com/item/public_token/exchange \
+  -H 'Content-Type: application/json' \
+  -d '{
+    "client_id": "[Plaid Client ID]",
+    "secret": "[Plaid secret]",
+    "public_token": "[Public token]"
+  }'
+  ```
+ Create a processor token for a specific account id.
+  
+  ```bash
+curl -X POST https://sandbox.plaid.com/processor/token/create \
+  -H 'Content-Type: application/json' \
+  -d '{
+    "client_id": "PLAID_CLIENT_ID",
```

Comment:
why is this not consistent with above (PLAID_CLIENT_ID vs [Plaid Client ID])?

Reply:
It is suppose to be consistent. Updated